scripts/hydra

```diff
@@ -193,7 +193,7 @@ install() {
   echo "Installing hydra ..."
   echo "Installing Framework..."
   create_template project $PROJECT_NAME
-
+  chmod -R 777 infra/docker/monitoring
```

Comment:
We don't need the 777 on CHMOD at this point, you can update to:
`chmod -R a+rwx ...your_directory`

Reply:
Yeah, and the rwx is equivalent to chmod 755, I don't know exactly why haha but they are different